src/devices_models/devices/common/timeseries_constraint.jl

```diff
@@ -1,57 +1,55 @@
 @doc raw"""
     device_timeseries_ub(psi_container::PSIContainer,
-                     ts_data::Vector{Tuple{String, Int64, Float64, Vector{Float64}}}
+                     ts_data::Dict{String,DeviceTimeSeries},
```

Comment:
The struct `DeviceTimeSeries` should contain the information in range data and the function should just take the struct with the required information. There is no need to have both structs as inputs.